### ISSUE TITLE
request: expose public interface for locating chrome path

### ISSUE THREAD

**Issue by knksmith57** [GoogleChrome/chrome-launcher#176]
The business logic used by `chrome-launcher` to locate the chrome binary is robust and well maintained. Unfortunately, it isn't exposed as part of the package's public interface.

Given `chrome-launcher` is effectively the authority for the best / most correct way to launch chrome, IMO, it makes sense to allow public access to this business logic instead of inadvertently encouraging others to attempt to reproduce its behavior.

In short, I'd like to expose a public interface for locating the chrome binary --essentially encapsulating [this logic from `src/chrome-launcher.ts`](https://github.com/GoogleChrome/chrome-launcher/blob/e67a10df/src/chrome-launcher.ts#L208-L213) in a discrete method and exporting it.

How do y'all feel about that?

**Comment by knksmith57** [comment]
how do y'all feel about this @patrickhulce / @benschwarz?

I'm happy to work + submit a PR if the feature is something the team is interested in.

**Comment by patrickhulce** [comment]
Exposing chrome finder as a function on our exports sounds like a good idea to me! :)
